node/bft/src/primary.rs

```diff
@@ -1114,7 +1119,7 @@ impl<N: Network> Primary<N> {
     /// Stores the certified batch and broadcasts it to all validators, returning the certificate.
     async fn store_and_broadcast_certificate(&self, proposal: &Proposal<N>, committee: &Committee<N>) -> Result<()> {
         // Create the batch certificate and transmissions.
-        let (certificate, transmissions) = proposal.to_certificate(committee)?;
+        let (certificate, transmissions) = tokio::task::block_in_place(|| proposal.to_certificate(committee))?;
```

Comment:
note: this is done via `block_in_place` instead of `spawn_blocking` in order to not have to clone the `Proposal`, which could get expensive enough to defeat the purpose